Game_Manual/Stats.md

```diff
@@ -29,31 +29,83 @@ How much your team members felt that you supported them in their learning. This
 
 How much your team members felt you contributed positively to the team culture. This is a weighted average with more recent projects counting more than earlier ones. Represented as a percentage (0-100%).
 
-## Discernment
+## Project Contribution
 
-Reflects the degree to which you are biased in your assessment of your contribution to projects (as compared to the average of others' assessment). Represented as a percentage corresponding to the average of how much you over- or under-estimate your contribution to projects.
+The amount that you contribute to a project determines a series of other stats, and also contains several sub-stats within.
 
-Zero is equal to perfect discernment (i.e. you estimate your contribution the same as your peers do). The closer this stat is to zero, the more accurate your discernment is.
+Within a given project, you have **actual** and **expected** contributions, as well as a **contribution gap**.
 
-#### Example
+To demonstrate how the various contribution stats are calculated, we'll use the following dataset in examples.
 
-Let's say you worked on three projects, with the following contribution rankings:
+#### Example Project Contribution Estimations
 
-| Project      | Contribution (self-estimated) | Contribution (team-estimated) |
-|:-------------|:------------------------------|:------------------------------|
-| #tiny-tigers | 20%                           | 30%                           |
-| #big-bees    | 50%                           | 45%                           |
-| #red-rabbits | 28%                           | 30%                           |
+| Project      | Team size | Self estimate | Team estimates |
+|:-------------|:----------|:--------------|:---------------|
+| #big-bees    | 3         | 50%           | 42%, 48%       |
+| #red-rabbits | 4         | 28%           | 27%, 35%, 28%  |
+| #tiny-tigers | 2         | 20%           | 30%            |
 
-Your discernment _per project_ is the difference between self- and team-estimated contribution as an absolute number. So the discernment stats for the above projects would be:
+### Actual Contribution
 
-| Project      | Discernment          |
+Your actual contribution is the average of your team's estimates of what percentage of the project you contributed to. It includes your own estimate of your contribution. Represented as a percentage (0-100%).
+
+In the above dataset, the player's actual contribution for the `#big-bees` project is the mean of _all_ the estimates, so it would be equal to 46.67% ( (50% + 42% + 48%) / 3 ).
+
+### Expected Contribution
+
+Your expected contribution how much you are expected to contribute to the project based on the team size so that each player contributes equally. Represented as a percentage (0-100%).
```

Comment:
Right now, no. It would change the nature of this stat. It represents expected contribution _before_ hours are calibrated, and assumes that the team will contribute equally regardless of hourly input.

If we make this a function of hours as well, then it changes the meaning of the stat. I'm not against it, but it is a change that we should probably spend some time on. New issue?